wire-ios/Wire-iOS Tests/TestPlans/GermanLocaleTests.xctestplan

```diff
@@ -0,0 +1,408 @@
+{
+  "configurations" : [
+    {
+      "id" : "9EE4E393-4136-4602-8B2B-6C76C7AE396A",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "commandLineArgumentEntries" : [
+
+    ],
+    "environmentVariableEntries" : [
+      {
+        "key" : "RECORDING_SNAPSHOTS",
+        "value" : "NO"
+      },
+      {
+        "key" : "SNAPSHOT_REFERENCE_DIR",
+        "value" : "$(SOURCE_ROOT)\/$(PROJECT_NAME) Tests\/ReferenceImages"
+      },
+      {
+        "key" : "FB_REFERENCE_IMAGE_DIR",
+        "value" : "$(SOURCE_ROOT)\/$(PROJECT_NAME) Tests\/ReferenceImages"
+      },
+      {
+        "key" : "IMAGE_DIFF_DIR",
+        "value" : "$(SOURCE_ROOT)\/SnapshotResults"
+      },
+      {
+        "key" : "SNAPSHOT_ARTIFACTS",
+        "value" : "$(SOURCE_ROOT)\/SnapshotResults"
+      }
+    ],
+    "language" : "de",
+    "region" : "DE",
+    "targetForVariableExpansion" : {
+      "containerPath" : "container:Wire-iOS.xcodeproj",
+      "identifier" : "8F42C537199244A700288E4D",
+      "name" : "Wire-iOS"
+    },
+    "testTimeoutsEnabled" : true
+  },
+  "testTargets" : [
+    {
+      "skippedTests" : [
```

Comment:
I think it would be better to have an explicit list of tests, rather than skipped tests.

Reply:
What do you mean? We're only running the DE tests here. I only selected those. Not sure why the rest are mentioned as skipped. We don't skip them though. They're not part of the TestPlan

Reply:
I think there's an option to automatically add new tests to the test plan. Check that is not enabled, that way only tests you explicitly select are in the plan, rather than explicitly selecting which tests are not in the plan.

Reply:
@johnxnguyen Done!

Reply:
![Screenshot 2023-09-21 at 16 58 36](https://github.com/wireapp/wire-ios-mono/assets/254198/f1df4117-948f-4c17-9e96-9002b7976e21)
on the test plan